static/app/views/eventsV2/table/quickContext.spec.tsx

```diff
@@ -589,6 +591,25 @@ describe('Quick Context', function () {
       ).toBeInTheDocument();
     });
 
+    it('Renders event id header', async () => {
+      jest.spyOn(ConfigStore, 'get').mockImplementation(() => null);
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/events/sentry:6b43e285de834ec5b5fe30d62d549b20/',
+        body: makeEvent({type: EventOrGroupType.ERROR, entries: []}),
+      });
+
+      delete defaultRow.title;
```

Comment:
Is this delete necessary for the test? I assume you may have added it to recreate the state where we have the "+" icon in the title, but I don't see that being tested here

Reply:
Yeah, don't need that there.